README.md

```diff
@@ -0,0 +1,6 @@
+1. Copy your msso_config.json file inside platforms/android/assets folder
+2. Copy the MAS libraries inside platforms/android/libs folder. Name the libraries as mentioned in the build.gradle file.
```

Comment:
Add the MAS libraries to the folder: platforms/android/libs. Name the libraries as shown in the build.gradle file.